src/pages/SignupV2.jsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Box,
+  Button,
+  Center,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  HStack,
+  Input,
+  SimpleGrid,
+  useToast,
+  VStack,
+} from '@chakra-ui/react';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import * as yup from 'yup';
+import { createUserInFirebase } from '../utils/firebaseAuthUtils';
+import logo from '../Assets/Logo.png'; // Ensure you have this asset
+
+const SignupV2 = () => {
+  return (
+    <>
+      <CreateAccount />
+    </>
+  );
+};
+
+const signupSchema = yup.object({
+  firstName: yup.string().required('Please enter your first name'),
+  lastName: yup.string().required('Please enter your last name'),
+  email: yup.string().email().required('Please enter your email address'),
+  password: yup
+    .string()
+    .required('Please enter your password')
+    .matches(
+      /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/,
+      'Password requires at least 8 characters consisting of at least 1 lowercase letter, 1 uppercase letter, 1 symbol, and 1 number',
+    ),
+  confirmPassword: yup
+    .string()
+    .required('Please re-enter your password')
+    .oneOf([yup.ref('password'), null], "Passwords don't match"),
+});
+
+const CreateAccount = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(signupSchema),
+    delayError: 750,
+  });
+
+  const toast = useToast();
+  const navigate = useNavigate();
+
+  const onSubmit = async (data) => {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'data' is defined but never used.